Sketch.java

```diff
@@ -23,14 +23,55 @@ public void setup() {
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	  //Creating the ground and the colour it will be 
+    ground();
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+    //Creating the sun 
+    sun();
```

Comment:
as per [Level 2 requirements](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-2), a demonstration of methods defined with parameters is required